securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -234,8 +234,7 @@ def _add_user(self, username, is_admin=False, hotp=None):
             hotp_secret.send_keys(hotp)
 
         if is_admin:
-            # TODO implement (checkbox is unchecked by default)
```

Comment:
raising a `NotImplementedError` seems like a reasonable change, someone should implement this when they need it in the tests

issue #3608